### ISSUE TITLE
Some audio stream don't work

### ISSUE THREAD

**Issue by MatteoNardi** [CastagnaIT/plugin.video.netflix#32]
*I'm submitting a ...*
  - [x] bug report
  - [ ] feature request
  - [ ] support request

### Addon version used

Current HEAD
da8ede27ded08a95

I reproduced it also with beta 23 and an older release.

### Description

Some audio stream don't work.

### Steps to Reproduce

1. Open "Chilling Adventures of Sabrina", first episode.
2. English audio stream doesn't work

In some episodes the english stream doesn't work, but it's not always the same.
Episode 1: english, italian, spanish not working
Episode 6: spanish and italian not working

These seem to change over time.

### Installation

* [ ] I'm using this Netflix Repo
* [ ] I'm using other Netflix Repo (Please specify which)
* [x] I'm using a different source: Github clone

### Operating System

* [x] Linux (x86/x64)
* [ ] OSX (x86/x64)
* [ ] Windows (x86/x64)
* [x] Linux (ARM)
* [ ] Android

I reproduced this with a Raspberry and with Archlinux x64.

## Debug log

https://pastebin.com/EGLNWhk6

## Other information

This started happening Saturday (the same episode was working the day before). I think something might have changed on the Netflix side...

**Comment by CastagnaIT** [comment]
i tried the episodes mentioned and other episodes, but to me everything language works as it should,
the errors in the logs seem to describe problems in audio streaming,
maybe they've already solved

**Comment by ASHJG** [comment]
hey i have the same issue although different for something different; Travelers season 2, episode 6,7,8, default audio stream is silent changing the audio stream to for example "audio description" works OK. I haven't seen this problem on any other programs. running 18.1, Netflix 14,23

**Comment by MatteoNardi** [comment]
For other reasons I'm switching account. On the new one the issue is not present anymore.
Old account is a "basic plan", the new one is a "premium plan". I don't know if the issue depends on the plan or some other account-setting.

**Comment by ASHJG** [comment]
I'm on the premium (4 devices / 4K)  not sure it's "plan" specific

**Comment by CastagnaIT** [comment]
I also have a premium account, so i've never had a problem like this before, 
in any case from the netflix addon i can not do anything, it concerns instead inputstream addon and/or kodi

**Comment by ASHJG** [comment]
Makes sense, I've just played the "silent" episodes ok on libreelec (RPI2), maybe its platform specific (Android?)

**Comment by CastagnaIT** [comment]
@ASHJG 
please put a log with debug infos
and the device model name

**Comment by CastagnaIT** [comment]
I'm collecting information, i thought the problema was limited to android 7 of certain devices,

but this on linux it changes the things, with the latest archlinux updates is the problem still there?

**Comment by cpetry** [comment]
I got this problem on Rpi2.
The default language (in my case german) is silent in some episodes whereas English works perfect.

**Comment by Smeulf** [comment]
@cpetry can you provide debug logs? Can you also post 'guisettings.xml' ?

The only problem I ever had here on RPi3 is only because passthrough was enabled and the receiver was not able to decode it. I'd like to see your full config.

Thanks.

**Comment by cpetry** [comment]
Here my guisettings:
[guisettings.zip](https://github.com/CastagnaIT/plugin.video.netflix/files/3273324/guisettings.zip)

Do I need to enable debug logging somewhere? Didn't find them in userdata

**Comment by Smeulf** [comment]
@cpetry yes please debug log it's important. You can enable it here in Kodi: Settings->System->Logging->Enable debug logging, and then try to play the movie/episode with the 'good' audio Trask then switch to the buggy audio track.

**Comment by CastagnaIT** [comment]
now you should no longer have problems, i removed the streams in question in favor of one with more quality, which will probably only be available for premium accounts
you can download the daily build

**Comment by CastagnaIT** [comment]
@cpetry when you have time please make a test and give a feedback thanks

**Comment by notoco** [comment]
I checked at home ... and did not even know that I had this problem too ...

**Comment by Fuzzillogic** [comment]
@CastagnaIT I had no audio on Jessica Jones S03E01 (Linux x86, both 720p and 1080p). Now that I've installed the the daily build it works just fine. Thanks!

**Comment by notoco** [comment]
Maybe I'll explain - still at 0.13 I had a problem with the Polish soundtrack in many movies. The problem appeared when the soundtrack was 2.0 and 5.1 - sometimes 2.0 did not want to work, 5.1 was OK. But there was always a problem with the volume level, and I will not play cartoons through AMP. So I set Dolby by definition and forgot about it. And since the last day, the daily works beautifully 2.0 :) I did not even know that I had this problem ...
